src/Components/AddMediaIcon/AddMediaIcon.module.css

```diff
@@ -0,0 +1,43 @@
+.addMediaIconContainer {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-direction: column;
+	height: min(20vh, 10vw);
+	width: min(20vh, 10vw);
+	min-width: 100px;
+	min-height: 100px;
+	border-radius: 5px;
+	margin: 10px;
+	overflow: hidden;
+	border-radius: 50%;
+  }
+
+.addfileButton {
+	margin: 20px;
+	width: max(12vw, 12vh);
+	height: max(12vw, 12vh);
+	border-radius: 50%;
+	border: 1px solid var(--translucent-white);
+	background-color: var(--semi-translucent-white);
+	cursor: pointer;
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+	text-align: center;
+	font-size: small;
+}
+
+.icon {
+	width: min(5vw, 5vh);
+	max-width: min(9vw, 9vh);
+	height: min(5vw, 5vh);
+	max-height: min(9vw, 9vh);
+	display: inline-block;
+	margin: 15px auto 0;
+}
+
+.addfileButton:hover {
+	border: 1px solid grey;
+	background-color: lightgrey;
```

Comment:
this should also be a variable

Reply:
default values from css like white, black grey should also be a variable?

Reply:
It's for consistency and reusability. 
For instance, if Jess comes along and says 'hmm, that grey should be lighter', you can change one variable to the new colour, knowing everything will be updated.